src/ignitus-StudentSignUp/Components/Signup.js

```diff
@@ -1,196 +1,181 @@
-                    </button>
-                  </div>
-                  <div className="or-seperator">
-                    <i className="text-black-50">or</i>
-                  </div>
-                  <div className="mb-4">
-                    <button type="button" className="btn btn-primary btn-rounded btn-linkedin mx-auto btn-block">
-                      <i className="fa fa-linkedin mr-3" />
+                  </button>
+                </div>
+                <div className="or-seperator">
+                  <i className="text-black-50">or</i>
+                </div>
+                <div className="mb-4">
+                  <button className="btn btn-primary btn-rounded btn-linkedin mx-auto btn-block">
+                    <i className="fa fa-linkedin mr-3" />
                     Linkedin
-                    </button>
-                  </div>
-                  <div className="text-center">
-                    <div>
-                  Already have an account?
-                      {' '}
-                      <Link to="/login/student" className="text-center linkform">
-                        {' '}
+                  </button>
+                </div>
+                <div className="text-center">
+                  <div>
+                  Already have an account?{" "}
+                  <Link to="/login/student" className="text-center linkform">
+                  {' '}
                    Log In
-                      </Link>
-                    </div>
-                  </div>
+                  </Link>
                 </div>
-              </form>
-            </div>
-          </div>
-          {success && (
-          <div className="alert alert-success alert-dismissible margin-Top">
-            <button type="button" className="close" data-dismiss="alert">&times;</button>
-            <strong>Success!</strong>
-            {' '}
-Please confirm your email address!
-          </div>
-          )}
-
-          {success === false && (
-          <div className="alert alert-success alert-dismissible margin-Top">
-            <button type="button" className="close" data-dismiss="alert">&times;</button>
-            {message}
-          </div>
-          )}
-
-          {emptymessage && (
-          <div className="alert alert-danger alert-dismissible margin-Top">
-            <button type="button" className="close" data-dismiss="alert">&times;</button>
-            <strong>Please!</strong>
-            {' '}
-fill the form!
-          </div>
-          )}
-
-          {equalmessage && (
-          <div className="alert alert-danger alert-dismissible margin-Top">
-            <button type="button" className="close" data-dismiss="alert">&times;</button>
-            <strong>Password </strong>
-            {' '}
-does not match the confirm password.!
+              </div>
+              </div>
+            </form>
           </div>
-          )}
         </div>
       </div>
+    </div>
     );
   }
 }
 
```

Comment:
Why did you removed these prop-types?

Reply:
I am sorry I didn't get you. I shifted the alert messages from there to support their current position.

Reply:
No, you removed the prop-types. Can you please add them again?